babel/src/main/codegen/config.fmpp

```diff
@@ -557,6 +557,10 @@ data: {
         "JSON_ARRAYAGG",
         "JSON_EXISTS",
         "JSON_VALUE",
+        "JSON_TYPE",
```

Comment:
Why 4 keywords added here?

Reply:
hi @zhztheplayer Because I have missed something that was not added before.

Reply:
I see, thanks for the explanation.

Reply:
And I see the format of non-reserved keyword list in babel parser is a little messy. I'll clean it a bit on top of your work.

Reply:
Well, thank you very much.